syndesis/plugins/org.fusesource.ide.syndesis.extension.ui/template-sources/step/javabean/pom.xml

```diff
@@ -93,6 +93,27 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-enforcer-plugin</artifactId>
+				<version>3.0.0-M2</version>
+				<executions>
+					<execution>
+						<id>enforce-maven</id>
+						<goals>
+							<goal>enforce</goal>
+						</goals>
+						<configuration>
+							<rules>
+								<requireMavenVersion>
+									<version>[3.5,3.6)</version>
+									<message>Maven needs to be a 3.5.x version. If you are using built-in default Maven in Eclipse, you need to configure a specific Maven as the default embedded version is 3.6+. See https://issues.jboss.org/browse/FUSETOOLS-3249 for more details.</message>
```

Comment:
See above, but I would change this message to: "The Maven version must be a 3.5.x version in Eclipse to work correctly. If using the default Maven version in Eclipse, which defaults to 3.6.x, you must configure a specific 3.5.x Maven instance manually. See https://issues.jboss.org/browse/FUSETOOLS-3249 for more details."